make install dependencies

Hello,

Not that I am sure about what I suppose to do, but as the start running `make install` was first asking for missing `gem` binary. And after gem installation, the Ruby package installation fails by asking for the root permissions:
```
Updating rubygems-update
ERROR:  While executing gem ... (Gem::FilePermissionError)
    You don't have write permissions for the /var/lib/gems/2.5.0 directory.
Makefile:47: recipe for target 'install' failed
make: *** [install] Error 1
```

--- a/Makefile
+++ b/Makefile
@@ -44,7 +44,7 @@ create-env: ## create conda environment
 
 ACTIVATE_ENV = source $(shell dirname $(dir $(CONDA)))/bin/activate $(CONDA_ENV)
 
-install: clean ## install dependencies
+install: clean create-env ## install dependencies
 	$(ACTIVATE_ENV) && \
 		gem update --system && \
 		gem install addressable:'2.5.2' jekyll:'< 4' jekyll-feed jekyll-environment-variables jekyll-github-metadata jekyll-scholar:'< 6' csl-styles awesome_bot html-proofer pkg-config kwalify

